docs/docs/Components/components-tools.md

```diff
@@ -31,238 +31,237 @@ For an example, see [Make any component a tool](/agents-tool-calling-agent-compo
+**Inputs**
 
-### Outputs
+| Name | Type | Description |
+|------|------|-------------|
+| Tool Name | String | The name used to reference the tool in the agent's prompt. |
+| Tool Description | String | A brief description of the tool to guide the model in using it. |
+| Keyspace | String | The name of the keyspace. |
+| Table Name | String | The name of the Astra DB CQL table to query. |
+| Token | SecretString | The authentication token for Astra DB. |
+| API Endpoint | String | The Astra DB API endpoint. |
+| Projection Fields | String | The attributes to return, separated by commas. Default: "*". |
+| Partition Keys | Dict | Required parameters that the model must fill to query the tool. |
+| Clustering Keys | Dict | Optional parameters the model can fill to refine the query. Required parameters should be marked with an exclamation mark (for example, `!customer_id`). |
+| Static Filters | Dict | Attribute-value pairs used to filter query results. |
+| Limit | String | The number of records to return. |
+
+**Outputs**
 
 | Name | Type | Description |
 |------|------|-------------|
 | Data | List[Data] | A list of [Data](/concepts-objects) objects containing the query results from the Astra DB CQL table. Each Data object contains the document fields specified by the projection fields. Limited by the `number_of_results` parameter. |
 | Tool | StructuredTool | A LangChain StructuredTool object that can be used in agent workflows. Contains the tool name, description, argument schema based on partition and clustering keys, and the query function. |
 
+</details>
+
 ## Bing Search API
 
 This component allows you to call the Bing Search API.
 
-### Inputs
-
-| Name                   | Type         | Description                           |
-|------------------------|--------------|---------------------------------------|
-| bing_subscription_key   | SecretString | Bing API subscription key             |
-| input_value            | String       | Search query input                    |
-| bing_search_url        | String       | Custom Bing Search URL (optional)    |
-| k                      | Integer      | Number of search results to return    |
-
-### Outputs
+<details>
+<summary>Parameters</summary>
 
-| Name    | Type      | Description                          |
-|---------|-----------|--------------------------------------|
-| results | List[Data]| List of search results               |
-| tool    | Tool      | Bing Search tool for use in LangChain|
+**Inputs**
 
-## Calculator Tool
-
-This component creates a tool for performing basic arithmetic operations on a given expression.
+| Name | Type | Description |
+|------|------|-------------|
+| bing_subscription_key | SecretString | Bing API subscription key |
+| input_value | String | Search query input |
+| bing_search_url | String | Custom Bing Search URL (optional) |
+| k | Integer | Number of search results to return |
```

Comment:
Add periods and articles to the descriptions. For example, "A Bing API subscription key.".